.github/workflows/check_pr.yml

```diff
@@ -118,10 +118,19 @@ jobs:
         with:
           node-version: 18
 
-      - name: Install dependencies
+      - name: Cache node modules
+        id: cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/docs/node_modules
+          key: node_modules-${{ matrix.os }}-${{ matrix.node }}-${{ hashFiles('**/npm-shrinkwrap.json') }}
```

Comment:
We're not using matrix for building docs, using as the key `node_modules` and the package-lock.json has is enough

Reply:
Fair point, was too focused on the caching story that I lost sight of the key structure and the matrix usage.